common/src/jni/main/cpp/conscrypt/native_crypto.cc

```diff
@@ -6877,6 +6877,25 @@ static jlong NativeCrypto_SSL_clear_options(JNIEnv* env, jclass, jlong ssl_addre
     return result;
 }
 
+static jint NativeCrypto_SSL_set_protocol_versions(JNIEnv* env, jclass, jlong ssl_address, CONSCRYPT_UNUSED jobject ssl_holder, jint min_version, jint max_version) {
```

Comment:
Any reason not to return `jboolean`? (Or do these wrappers typically return `jint`?)

The only reason we write `int` everywhere is because C. Most of the time we mean `bool`.

Reply:
We've historically returned jint, and I'd rather keep consistent with that for the time being.  Changing it boolean is a sensible thing to do in the long term, though.